src/engines/nwn2/roster.h

```diff
@@ -40,13 +40,19 @@ class Roster {
 	Roster();
 	~Roster();
 
+	/** Cycle through the roster members */
+	Common::UString getFirstRosterMember();
+	Common::UString getNextRosterMember();
+
 private:
 	struct Member {
 		Common::UString rosterName; ///< Unique roster name.
 	};
 
 	std::list<Member> _members; ///< List of roster member instances.
 
+	uint32 _lastRetrieved; ///< Index of last retrieved member.
```

Comment:
You're not initializing `_lastRetrieved` anywhere, so calling `getNextRosterMember()` would invoke UB. You should maybe rather set it to some kind of guard value, like `UINT32_MAX`, and have `getNextRosterMember()` check and throw and error.

(See also my comment in that method)